lib/rustler_precompiled.ex

```diff
@@ -263,7 +272,7 @@ defmodule RustlerPrecompiled do
   @native_dir "priv/native"
 
   @doc """
-  Returns URLs for NIFs based on its module name.
+  Returns tuples of file names and their URLs for NIFs based on its module name.
```

Comment:
This is a backwards incompatible change in the public API. If you wish, I can rename this (and `current_target_nif_urls `) to keep the public API unchanged.

Reply:
Yeah, I think we can have two versions, and deprecate the current one. WDYT?